package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "module_4",
```

Comment:
It's because of the quotes around the property. If you want to lint json files, you'll need a json eslint plugin.